### ISSUE TITLE
Saved Queries in QuestDB Web Console

### ISSUE THREAD

**Issue by EmmettM** [questdb/questdb#2960]
### Describe the solution you'd like.

It would be nice to be able to save queries that I run often in QDB Web Console. I have to retype them every time to run.

**Comment by bluestreak01** [comment]
hey Emmet, is this request related to the Web Console?

**Comment by EmmettM** [comment]
> hey Emmett, is this request related to the Web Console?

Yes apologies for not being clearer– I will change the subject from Cloud to Web Console.
